scripts/Interface/ui/sans_isis/beam_centre.py

```diff
@@ -0,0 +1,171 @@
+from __future__ import (absolute_import, division, print_function)
```

Comment:
is this a view, presenter or model?

Reply:
This is a view. Generally the views are in the ui/sans_isis folder and the model and presenters are in SANS/sans/gui_logic